src/core/diff/strategies/multi-search-replace.ts

```diff
@@ -367,7 +368,6 @@ Only use a single line of '=======' between search and replacement content, beca
 		const replacements = matches
 			.map((match) => ({
 				startLine: Number(match[2] ?? 0),
-				endLine: Number(match[4] ?? resultLines.length),
```

Comment:
We're not using the passed in endLine anymore (just taking the start line plus to length of the search block), but I figure we can clean that up as a follow-up if this doesn't cause any unforeseen problems.

Reply:
#2615